src/test/kotlin/org/elm/ide/typing/ElmOnEnterSmartIndentHandlerTest.kt

```diff
@@ -65,13 +65,13 @@ f =
     fun `test after equals in let declaration partial`() = doTest("""
 f =
     let
-        x ={-caret-}
+        x = {-caret-}
```

Comment:
So this test was actually broken. The problem seems to be that the end decl token from the `in` now comsumes the newline that is generated when you type `\n`, so IntelliJ thinks you're lined up with the `let`, rather than the `x`. I don't know of a good way to fix this without adding extra whitespace, which I did here.

Reply:
I have fixed this on my side. The problem was in `ElmOnEnterSmartIndentHandler`. I'm going to merge your PR and then apply my fix.